style.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+* {
+  font-family: "Poppins", sans-serif;
+}
+
+header {
+  background-color: #fff;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  width: 100%;
+  height: 48px;
+  margin-left: 6px;
+  margin-top: -7px;
+  gap: 38%;
+}
+
+.logo {
+  display: flex;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  text-align: center;
+  color: #6070ff;
+  width: 76px;
+  height: 20px;
+  text-decoration: none;
+  align-items: center;
+}
+
+.humbourger {
+  height: 12px;
+  width: 18px;
+  left: 3px;
+  top: 6px;
+  color: rgba(96, 112, 255, 1);
+}
+
+.app-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  background-image: url("media/header-image.png");
+  background-color: rebeccapurple;
+  width: 100%;
+  height: 48px;
+}
+
+.menu {
+  display: none !important;
+}
+
+.header-ambony {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 100%;
+  background-color: #fff;
+  background-image: url(media/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  top: 2.875rem;
+  border-bottom-left-radius: 60px;
+  height: 44.88rem;
+}
+
+.narson {
+  display: flex;
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 60px;
+  letter-spacing: 0.3700000047683716px;
+  align-items: center;
+  width: 327px;
+  height: 9.37rem;
+  margin: 20px;
+  color: #091e42;
+  font-style: bold;
+  order: 0;
+}
+
+.detail {
+  font-size: 20px;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 28px;
+  text-align: left;
+  color: rgba(52, 69, 99, 1);
+  margin: 0 20px;
+  width: 327px;
```

Comment:
[Optional] I would suggest removing the fixed width values as we have to make our designs responsive, you can use percentage values if needed